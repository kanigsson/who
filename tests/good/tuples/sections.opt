--clear --tuples --transform-only --noprelude --input-annot
