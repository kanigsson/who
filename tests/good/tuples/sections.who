section basiclogic coq predefined
  pangoline predefined
  logic \/  : prop -> prop -> prop
end
logic int_min  : int -> int -> int
axiom int_min_is_some : True \/ True
