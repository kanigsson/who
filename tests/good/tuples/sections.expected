logic \/  : prop -> prop -> prop
logic int_min  : int -> int -> int
axiom int_min_is_some: True 
